src/test/resources/logInUser.properties

```diff
@@ -0,0 +1,2 @@
+email = 17141@rambler.ru
+password = 12345678
```

Comment:
Maybe this data can be stored in file "user-information-test-date.properties" where is stored this information.